### ISSUE TITLE
Proto Options not working

### ISSUE THREAD

**Issue by tbillington** [stanley-cheung/Protobuf-PHP#12]
I'm trying to use the options in the proto files as specified [here](http://drslump.github.io/Protobuf-PHP/protobuf-php.5.html), but they are causing errors. `libprotoc 3.0.1`.

``` proto
syntax = "proto3";

package tutorial.values;

option (php.suffix) = ".pb.php";
option (php.namespace) = "Tutorial.Values";

message Value {
  string value = 1;
}
```

Error: `Option "(php.suffix)" unknown.`

**Comment by stanley-cheung** [comment]
Proto3 is not supported in this fork. Please see the official google/protobuf support soon.

**Comment by tbillington** [comment]
grpc website still points to this repo for proto 3.1.0, I don't quite understand why if it doesn't support proto3? http://www.grpc.io/docs/quickstart/php.html#install-grpc

**Comment by stanley-cheung** [comment]
We haven't released grpc version 1.1 yet, which by that time will support protobuf 3.1.0
